Modelica/Magnetic/QuasiStatic/FundamentalWave/Components/Crossing.mo

```diff
@@ -9,11 +9,6 @@ model Crossing "Crossing of connections"
     annotation (Placement(transformation(extent={{90,-110},{110,-90}})));
   Interfaces.NegativeMagneticPort port_n2 "Connected with port_n1"
     annotation (Placement(transformation(extent={{90,90},{110,110}})));
-equation
-  connect(port_p1, port_p2) annotation (Line(
-      points={{-100,100},{-100,20},{0,20},{0,-20},{100,-20},{100,-100}}, color={255,170,85}));
-  connect(port_n2, port_n1) annotation (Line(
-      points={{100,100},{100,0},{-100,0},{-100,-100}}, color={255,170,85}));
```

Comment:
See previous comment on crossing.

Reply:
Re-introduce connections in 217f0dae87abb7a6bdac938aa142a4185f348d50